### ISSUE TITLE
[BUG] <title>v2版本无法启动

### ISSUE THREAD

**Issue by qazxswaaa** [netease-youdao/QAnything#497]
### 是否已有关于该错误的issue或讨论？ | Is there an existing issue / discussion for this?

- [X] 我已经搜索过已有的issues和讨论 | I have searched the existing issues / discussions

### 该问题是否在FAQ中有解答？ | Is there an existing answer for this in FAQ?

- [X] 我已经搜索过FAQ | I have searched FAQ

### 当前行为 | Current Behavior

在dockers desktop中下载了相关镜像文件，命令行docker-compose -f docker-compose-win.yaml up启动后一直不行 
✔ Container qanything-container-local  Stopped                                                                                                                                                             0.0s
 - Container mysql-container-local      Stopping                                                                                                                                                            0.1s
 - Container milvus-standalone-local    Stopping                                                                                                                                                            0.1s
 ✔ Container es-container-local         Stopped                                                                                                                                                             0.0s
dependency failed to start: container es-container-local is unhealthy

### 运行环境 | Environment

```Markdown
- OS:
- NVIDIA Driver:
- CUDA:
- Docker Compose:
- NVIDIA GPU Memory:
```

**Comment by houzy** [comment]
我也报相同的错误

**Comment by dd404x** [comment]
1

**Comment by JOMOelicit** [comment]
你好请问解决了吗？

**Comment by Jason-cs18** [comment]
一般重新启动docker-compose即可

**Comment by zyb8543d** [comment]
类似的问题请问大家怎么解决的，完全按照说明文档来的，还报错。
```
milvus-standalone-local    | [2024/11/05 07:49:22.567 +00:00] [WARN] [grpcclient/client.go:249] ["failed to get client address"] [error="find no available datacoord, check datacoord state"]
milvus-standalone-local    | [2024/11/05 07:49:22.567 +00:00] [WARN] [grpcclient/client.go:464] ["fail to get grpc client in the retry state"] [client_role=datacoord] [error="find no available datacoord, check datacoord state"]
milvus-standalone-local    | [2024/11/05 07:49:22.567 +00:00] [WARN] [grpcclient/client.go:249] ["failed to get client address"] [error="find no available querycoord, check querycoord state"]
milvus-standalone-local    | [2024/11/05 07:49:22.567 +00:00] [WARN] [grpcclient/client.go:464] ["fail to get grpc client in the retry state"] [client_role=querycoord] [error="find no available querycoord, check querycoord state"]
dependency failed to start: container milvus-standalone-local is unhealthy
Gracefully stopping... (press Ctrl+C again to force)

```

**Comment by Heisenbergxh** [comment]
请问有解决该问题么

**Comment by charleshuangruo** [comment]
同样的问题，在docker里提示qanything-container-local容器错误，无法启动。其他的容器都正常启动：Ports are not available: exposing port TCP 0.0.0.0:8777 -> 0.0.0.0:0: listen tcp4 0.0.0.0:8777: bind: Only one usage of each socket address (protocol/network address/port) is normally permitted.

**Comment by GreatStep** [comment]
至今，没有解决~~
